### PR TITLE
added a p tag

### PR DESCRIPTION
**Pull request by alvinwchou**
- added a p tag

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,5 +8,7 @@
 </head>
 <body>
     <h1>Riley Fan Page</h1>
+
+    <p>How is your guinea pig doing?</p>
```
Comment (wil828) [inline index.html]: She's doing well! Thanks for asking. She's a grumpy old lady and keeps fighting with the other pigs haha.

**Review by wil828 — CHANGES_REQUESTED** [review]
Solid one liner.

File: index.html (hunk 1 above)
Comment (alvinwchou) [reply index.html]: Grumpy old ladies can be like that  lol

**Review by wil828 — APPROVED** [review]
Much Wow. Approved. Need Merge

**Pull request merged**